### PR TITLE
Feature: Added support for displaying background images in folders

### LINKED ISSUE #13653: Feature: Support for changing folder background image
### What feature or improvement do you think would benefit Files?

Please add the folder background feature like Windows XP.
image
```ini
[{BE098140-A513-11D0-A3A4-00C04FD706EC}]
Attributes=1
IconArea_Image=bliss.jpg
IconArea_Text=0x00808080
```

This feature exists on MacOS:
https://youtu.be/Er8dU6A6hYU?si=gMV_BFQt1gksgu8i

This feature is good for Music folders:

### Requirements

- This proposal will accomplish X
- This proposal will accomplish Y
- This proposal will accomplish Z

### Files Version

2.0.36.0

### Windows Version

10.0.22621.2428

### Comments

_No response_

### PR DESCRIPTION
**Pull request by yaira2**
**Resolved / Related Issues**

To prevent extra work, all changes to the Files codebase must link to an approved issue marked as `Ready to build`. Please insert the issue number following the hashtag with the issue number that this Pull Request resolves.
- Closes #13653

**Steps used to test these changes**

Stability is a top priority for Files and all changes are required to go through testing before being merged into the repo. Please include a list of steps that you used to test this PR.

1. Opened Files ...
2. ...

### CODE REVIEW

File: src/Files.App/Data/Models/ItemViewModel.cs [hunk 1]
```diff
@@ -1750,6 +1787,60 @@ private void CheckForSolutionFile()
 				.FirstOrDefault()?.ItemPath;
 		}
 
+		public void CheckForBackgroundImage()
```
Comment (yaira2) [inline src/Files.App/Data/Models/ItemViewModel.cs:1790]: I'm thinking of creating a dedicated helper for this.

**Comment by yaira2** [comment]
This PR is ready for review, but won't be merged until after v3.5.

**Comment by yaira2** [comment]
@hishitetsu I disabled custom backgrounds in Columns View.

File: src/Files.App/ViewModels/ShellViewModel.cs [hunk 2]
```diff
@@ -1751,6 +1790,56 @@ private void CheckForSolutionFile()
...
+				return;
+
+			// Read data
+			var lines = File.ReadLines(iniPath);
+			var keys = lines.Select(line => line.Split('='))
+								.Where(parts => parts.Length == 2)
+								.ToDictionary(parts => parts[0].Trim(), parts => parts[1].Trim());
+
+			// Image source
+			if (folderSettings.LayoutMode is not FolderLayoutModes.ColumnView && keys.TryGetValue("Files_BackgroundImage", out var backgroundImage))
```
Comment (hishitetsu) [inline src/Files.App/ViewModels/ShellViewModel.cs:1806]: If I switch to the column layout with an image in the background, the image does not disappear.
Similarly, if I switch from the column layout to another layout, the image does not appear.

Comment (yaira2) [reply src/Files.App/ViewModels/ShellViewModel.cs:1806]: I think this is because we only run this code when loading items

Comment (hishitetsu) [reply src/Files.App/ViewModels/ShellViewModel.cs:1806]: We can add a `LayoutModeChangeRequested` event.

Comment (yaira2) [reply src/Files.App/ViewModels/ShellViewModel.cs:1812]: Added

**Review by hishitetsu — APPROVED** [review]
Parsing of INI files needs to be improved, such as considering sections, but that can be addressed in #15654.
LGTM

**Pull request merged**